internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -395,8 +407,138 @@ func mutatingToWebhookDescription(webhook admissionregv1.MutatingWebhook, depNam
 	return description
 }
 
+// The default AdmissionReviewVersions set in a CSV for conversion webhooks
+// if ConversionReviewVersions is not specified in CRDs.
+var defaultConversionReviewVersions = []string{"v1beta1", "v1"}
+
+// conversionToWebhookDescription builds a webhookDescription for conversion webhook with the
+// WebhookClientConfig specified in either v1 or v1beta1 CRDs.
+func conversionToWebhookDescription(crdNames []string, v1WebhookClientConfig *apiextv1.WebhookConversion, v1beta1WebhookClientConfig *apiextv1beta1.CustomResourceConversion, depName string, ws *corev1.Service) operatorsv1alpha1.WebhookDescription {
+
+	conversionReviewVersions, whPath, serviceRefName, serviceRefPort := getConversionWebhookDetails(v1WebhookClientConfig, v1beta1WebhookClientConfig)
+
+	description := operatorsv1alpha1.WebhookDescription{
+		Type:                    operatorsv1alpha1.ConversionWebhook,
+		ConversionCRDs:          crdNames,
+		AdmissionReviewVersions: conversionReviewVersions,
+		WebhookPath:             &whPath,
+		DeploymentName:          depName,
+		SideEffects: func() *admissionregv1.SideEffectClass {
```

Comment:
Though `SideEffects` is not required by olm while [setting up conversion webhook](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/install/webhook.go#L186), a null value creates error.